src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -63,6 +63,25 @@ The API used to retrieve the value is:
   The number of times garbage collection has occurred for the specified generation
 of objects.
 
+#### process.runtime.dotnet.**gc.objects.size**
+
+Count of bytes currently in use by objects in the GC heap that haven't been
+collected yet.
+Fragmentation and other GC committed memory pools are excluded.
+The value is available irrespective of GC events.
```

Comment:
Not very clear statement.

Reply:
Updated again in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/772/commits/12919a61ac716f38e7895c0e3d8304895e1839be. 
I tried to be consistent with other metrics description.
Indication that value is not a snapshot is included in description of API used.